.github/workflows/create.yml

```diff
@@ -25,9 +25,21 @@ jobs:
             cibw_skip: '*-win32'
           - os: ubuntu-latest
             architecture: 'x64'
-            cibw_skip: '*-manylinux_i686'
+            cibw_archs: 'x86_64'
+          - os: ubuntu-latest
+            architecture: 'x86'
+            cibw_archs: 'i686'
```

Comment:
We want to skip the build for `*-manylinux_i686` wheels, as we removed support in a previous version.
Can you please change the config to only build `x86_64` and `aarch64` wheels?

Reply:
Sure. Will remove i686 (x86).